app/controllers/api/messaging.php

```diff
@@ -1216,11 +1564,12 @@
     ->param('content', '', new Text(64230), 'Email Content.')
     ->param('status', 'processing', new WhiteList(['draft', 'processing']), 'Message Status.', true)
```

Comment:
Let's add in the description that the value must be one of either draft or status

Reply:
done this